pkgs/applications/misc/dwmblocks/default.nix

```diff
@@ -2,7 +2,7 @@
 
 stdenv.mkDerivation {
   pname = "dwmblocks";
-  version = "unstable-2020-12-27";
+  version = "0-unstable-2020-12-27";
```

Comment:
There have also been new commits in the meantime. It might be good to update the package.

Reply:
Done